meeting_minutes/2024/2024-01-31.md

```diff
@@ -0,0 +1,177 @@
+    - currently done by CISA VEX WG
+    - goal: get an overview how VEX and machine-readable security advisories are used today
+    - each CSAF answer (even if you are just planning to use it) helps
+    - Justin adds details on the questionnaire
+    - Thomas Schmidt: February, 29 is the deadline
+  - Managed service providers for CSAF trusted providers:
+    - at least 2 organizations offer a CSAF trusted provider as managed service:
+      - [CERT@VDE](https://cert.vde.com/en/more/hostedcsafprovider/)
+      - [Intevation](https://intevation.de/index.en.html) (last paragraph)
+    - Any other offers known?
+  - New tools:
+    - [CSAF walker](https://github.com/ctron/csaf-walker)
+      - RUST implementation of a tool to retrieve and verify CSAF documents
+      - can be used as library
+      - already used in production, needs more help to implement all functions
+    - [Trivy with CSAF support](https://github.com/aquasecurity/trivy/pull/5535):
+      - contributed by Bitnami
+      - Should be available with next version `0.49.0`
+      - currently working of the [implementation of relationships](https://github.com/aquasecurity/trivy/pull/5923/files)
+      - should be added to [tool list](https://github.com/oasis-open/csaf-documentation/issues/50)
+    - [ISDuBA (WIP)](https://github.com/ISDuBA/ISDuBA)
+      - just started
+      - will be implementation of a [CSAF management system](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#9112-conformance-clause-12-csaf-management-system)
+  - CSAF Workshops in Munich (Germany) 2023-12-12 - 2023-12-15:
+    - series of 3 workshops:
+      - CSAF boot camp
+      - CSAF writer's guild
+      - CSAF distribution
+    - 2 ask the expert sessions
+    - jointly done by CISA and BSI
+    - in general: positive feedback
+    - next time: more advertising in advance
+    - Justin reports that the workshops were great and the group was very motivated and it is plausible the participants will implement the learnings in their work field
+    - Justin reports on ideas to grow the success of the workshops and how to increase the frequency of such workshops 
+    - Thomas Schaffer asks if maybe online on-demand trainings could be offered or did such a discussion already happen in the past
+    - Thomas Schmidt reports that all three workshops were recorded and currently the post production is working on publishing the material
+- Idea:
+  - Community days:
+    - issue: exchange in the community is extremely important (networking, help with tools, convincing management)
+    - idea: community members host small (1 or 2 day) events
+    - content:
+      - presentations
+      - success stories
+      - new tools
+      - user feedback session
+      - networking
+  - Jane reports that the Cyber Security Alliance is planning a plug fest - in case of interest please contact Jan per email and the date is this year April, 10 - 12
```

Comment:
```suggestion
  - Jane reports that the Cyber Security Alliance is planning a plug fest (near Washington DC area) - in case of interest please contact Jane per email and the date is this year April, 10 - 12